server/potentialMatch.py

```diff
@@ -16,28 +16,27 @@ def findPotentialMatches(id):
             
             genderPreference = getGenderPreference(id, cursor, connection)
             if genderPreference == "":
-                return {"response": "No Matches", "id": id, "Number of Matches": 0}
+                return {"response": "No preferred gender specified by user", "id": id, "Number of Matches": 0}
 
             existingIds = findExistingMatches(cursor, id)
             potentialIds = getPotentialIds(id, genderPreference, cursor, connection)
             potentialIds = [i for i in potentialIds if i not in existingIds]
             if len(potentialIds) == 0:
-                return {"response": "No Matches", "id": id, "Number of Matches": 0}
+                return {"response": "No potential matches using preferred gender", "id": id, "Number of Matches": 0}
```

Comment:
I think these should be returned as success. I dont' think they should have to have matches to submit the questionnaire

Reply:
should i also change the response for lines 27-29 to success?
            responses = getQuestionnaireResponses(potentialIds, cursor, connection)
            if len(responses) == 0:
                return {"response": "No potential matches using questionnaire responses", "id": id, "Number of Matches": 0}

Reply:
Yeah because otherwise the user will be stuck at the questionnaire page. I don't think them having any matches is an error